types.ts

```diff
@@ -970,6 +977,45 @@ export type VariousSupportedAuthentication =
  */
 export type VariousSupportedAuthenticationTypes = $Values<Pick<VariousSupportedAuthentication, 'type'>>;
 
+/**
+ * @hidden
+ */
+export enum ReservedAuthenticationKeys {
+  /**
+   * References the default user authentication of the pack.
+   */
+  Default = 'DEFAULT',
+
+  /**
+   * References the system authentication of the pack.
+   */
+  System = 'SYSTEM',
+}
+
+/**
+ * TODO(patrick): Unhide this.
+ * @hidden
+ */
+export interface AuxiliaryAuthentication {
+  authentication: StandardAuthentication;
+
+  /**
+   * Passed into formula context.
+   * Cannot be one of the ReservedAuthenticationKeys.
+   */
+  key: string;
+
+  displayName: string;
+
+  description?: string;
+
+  /**
+   * If true, this authentication can be used to sync permissions associated with data
+   * in addition to the data itself.
+   */
+  canSyncPermissions?: boolean;
```

Comment:
Is this meant to capture the idea of "supports meta syncs"? If so, I don't know if it's quite sufficient. A personal access token _could_ sync permissions, but it couldn't fetch data from the entire org.

Reply:
If I rename AuxiliaryAuth=>AdminAuth, does that fix this?

Reply:
What did we decide to call meta ingestions in product? We probably want to align with that. I think some bit here is necessary so that we know we can/should offer this as an option for the meta fork in the setup UX.

Reply:
Seems like https://coda.io/d/Brain-Library-Hub-go-brain-library_dotKT55OqGX/Proposal-Naming-of-connections_suR3ShOn is saying we will have "organization" connections that "mirror permissions". Maybe I'm over-indexing on the `permissionSyncMode` that I added before, but I feel like `canSyncPermissions` still works with this framing just fine.